src/models/Accommodation.js

```diff
@@ -74,15 +78,15 @@ const AccommodationDefinition = (sequelize, DataTypes) => {
   // eslint-disable-next-line func-names
   Accommodation.findWhereStartedBy = function (startedBy) {
     const accommodation = this;
-    return accommodation.find({
+    return accommodation.findAll({
       where: {
         startedBy
       }
     });
   };
 
   // eslint-disable-next-line func-names
-  Accommodation.rescind = function (bookedBy) {
+  Accommodation.rescind = function (bookedBy, id) {
```

Comment:
Woof! Use ES6 arrow functions.
Apparently, hound has stopped doing it's job.

Reply:
Please check comment to previous review.